doc/svc.md

```diff
@@ -42,14 +42,97 @@ frame). This mode of operation has several implications.
 
 First, it means that if the SFU decides to skip a frame for whatever reason,
 it cannot go back and change that decision because there would be no space left
-in the sequence numbers. So decisions about if and how to forward a frame apply
+in the sequence numbers.  Furthermore, if packet re-ordering or loss has occurred,
+a decision must be made immediately as to how much of a gap to leave in the
+sequence numbers so delayed packets have a place to be transmitted.
+
+When a frame arrives and it is to be forwarded, the SFU needs to determine how to
+assign its projected sequence numbers.  Decisions on how to assign sequence numbers
+are done based on _new_ and the _top_ frames respectively, where the top frame is the
+already-received frame that has the highest sequence number and new frames are frames
+subsequent to the top in sequence number space.
+
+If there is a gap in receive sequence numbers between the top and a new frame,
+the SFU can determine the worst-case gap size of projected sequence numbers to
+leave such that any not-yet-received packets can be forwarded correctly.
+However, the SFU can't (in general) know the temporal layer of frames it has
+not yet received. If it turns out that the not-yet-received frames were in fact
+not ones that the SFU wanted to forward, this will result in sequence number
+gaps in the projected bitstream, which the receiver may interpret as packet
+loss.  (Care must be taken in bandwidth estimation algorithms not to incorrectly
+reduce bandwidth based on this "false" packet loss.)
```

Comment:
For posterity, maybe we should point out here that when we're doing remb-based bwe, packet loss needs to be ignored (which is what we agreed upon) because we don't have a reliable way for computing loss?

Reply:
I updated this to say that.